packages/commonwealth/client/styles/index.scss

```diff
@@ -10,6 +10,9 @@ body {
   box-sizing: border-box;
   height: 100vh;
   width: 100%;
+  //Prevents overscroll on macOS
```

Comment:
This was added in order to prevent over-scrolling mainly used on Mac. The more long term solution standardized padding/margin across the app which is in the roadmap.